lib/pliny/tasks/db.rake

```diff
@@ -34,7 +34,7 @@ begin
     task :seed do
       if File.exist?('./db/seeds.rb')
         database_urls.each do |database_url|
-          Sequel.connect(database_url)
+          DB = Sequel.connect(database_url)
```

Comment:
Hey! Getting back to this.

So I just silenced the const warning. Not pretty, but it's code in Pliny exactly to make sure the Pliny app's code is not ugly :}